ports/llvm/vcpkg.json

```
@@ -362,14 +359,18 @@
             "target-ve",
             "target-webassembly",
             "target-x86",
-            "target-xcore"
+            "target-xcore",
+            "target-xtensa"
           ]
         }
       ]
     },
     "target-amdgpu": {
       "description": "Build with AMDGPU backend."
     },
+    "target-arc": {
+      "description": "Build with ARC backend (experemental)."
```

```suggestion
      "description": "Build with ARC backend (experimental)."
```
Similar in other spots

---

Fixed.